openedx/adg/admin_dashboard/applications/helpers.py

```diff
@@ -117,20 +117,25 @@ def get_table_date_format(date):
     return get_formatted_date(date, '%d/%m/%Y')
 
 
-def get_date_and_course_status_ordering(application_type):
+def get_date_and_course_status_ordering(application_type, order_by):
     """
     Return `date_accepted` as the date priority for accepted applications and `date_submitted` for other
     type of applications. Also returns ordering on the basis of course statuses i.e first Completed,
     then In Progress and finally Not Started
 
     Arguments:
         application_type (str): application status
+        order_by (str): indicates whether to sort applications submission/acceptation date in ascending or descending
 
     Returns:
         tuple: A tuple of str values
     """
+    order_by = '-' if order_by == DESCENDING else ''
+
     date_to_sort_applications = (
-        '-modified' if application_type == UserApplication.ACCEPTED else '-user__application_hub__submission_date'
+        f'{order_by}modified'
+        if application_type == UserApplication.ACCEPTED
+        else f'{order_by}user__application_hub__submission_date'
     )
```

Comment:
Why don't we simply pretend the order_by value instead of using f strings. '-' for desc and '' for ascending.
```
date_to_sort_applications = 'modified' if application_type == UserApplication.ACCEPTED else 'user__application_hub__submission_date'
date_to_sort_applications = order_by + date_to_sort_applications
```